addons/product/views/product_attribute_views.xml

```diff
@@ -103,6 +117,7 @@
                 <field name="html_color" invisible="display_type != 'color' or image" widget="color"/>
                 <field name="image" invisible="display_type != 'color'" widget="image" options="{'size': [70, 70]}"/>
                 <field name="ptav_active" optional="hide"/>
+                <field name="default_extra_price" readonly="1"/> 
```

Comment:
```suggestion
                <field name="default_extra_price"/> 
```
No need to add readony as related fields are already readonly :slightly_smiling_face: